Tests/MapboxDirectionsTests/TransportTypeTests.swift

```diff
@@ -0,0 +1,52 @@
+import XCTest
+@testable import MapboxDirections
+
+class TransportTypeTests: XCTestCase {
+    func testDecoding() {
+        let examples: [Example<String, TransportType>] = [
+            Example("driving", .automobile),
+            Example("ferry", .ferry),
+            Example("movable bridge", .movableBridge),
+            Example("unaccessible", .inaccessible),
+            Example("walking", .walking),
+            Example("pushing bike", .walking),
+            Example("cycling", .cycling),
+            Example("train", .train)
+        ]
+
+        let decoder = JSONDecoder()
+
+        for example in examples {
+            let decoded = try? decoder.decode(
+                CodableContainer<TransportType>.self,
+                from: jsonData(type: example.input)
+            )
+
+            XCTAssertEqual(decoded?.wrapped, example.expected)
+        }
+    }
+
+    private func jsonData(type: String) -> Data {
+        """
```

Comment:
This appears to be the cause of a build failure on CI:

```
/Users/distiller/project/Tests/MapboxDirectionsTests/TransportTypeTests.swift:34:31: warning: expression of type 'Data' is unused
        """.data(using: .utf8)!
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
/Users/distiller/project/Tests/MapboxDirectionsTests/TransportTypeTests.swift:35:5: error: missing return in a function expected to return 'Data'
    }
    ^
```

```suggestion
        return """
```

Reply:
I already got used to omitting return statements in Swift 5.1 😅

Reply:
We’ll get there before long! 😄